recipes/openimageio/all/patches/2.1.18.1.patch

```diff
@@ -0,0 +1,198 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index fbf52165..dc4f16d0 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -171,7 +171,7 @@ if (OIIO_BUILD_TOOLS AND NOT BUILD_OIIOUTIL_ONLY)
+     add_subdirectory (src/iinfo)
+     add_subdirectory (src/maketx)
+     add_subdirectory (src/oiiotool)
+-    add_subdirectory (src/testtex)
++    # add_subdirectory (src/testtex)
+     add_subdirectory (src/iv)
+ endif ()
+ 
+@@ -477,7 +477,7 @@ if (WIN32)
+     set(CPACK_PACKAGE_EXECUTABLES "iv" "iv - Image Viewer")
+ #    set(CPACK_CREATE_DESCTOP_LINKS "iv" "iv - Image Viewer")
+     set(CPACK_NSIS_MODIFY_PATH ON)
+-    include (InstallRequiredSystemLibraries)
++    # include (InstallRequiredSystemLibraries)
```

Comment:
is it needed to patch? e.g. setting `CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP` should skip it I suppose

Reply:
I checked [InstallRequiredSystemLibraries.cmake](https://github.com/Kitware/CMake/blob/5170263c3fc8ac5fdbb880cb8a1eaec48ec36eda/Modules/InstallRequiredSystemLibraries.cmake#L748), that variable only affects "additional runtime libraries". Runtime libs are copied unconditionally, the only fix is to comment this line.

Reply:
if `CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON` than all the `install(...)` directives are skipped. I would side with SSE4.